docs/development/release.md

```diff
@@ -8,6 +8,17 @@ The kops project is released on an as-needed basis. The process is as follows:
 1. The release issue is closed
 1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] kops $VERSION is released`
 
+## Branches
+
+We maintain a `release-1.4` branch for kops 1.4.X, `release-1.5` for kops 1.5.X etc.
+
+We create new branches from master as a new kops version is released (or in preparation for the release).
+
+Generally we don't encourage users to run older kops versions, or older branches, because kops should remain
+compatible with older versions of kubernetes.  But certain when a new major version of kops is released,
```

Comment:
`kubernetes` should be capitalized. Second sentence needs to be cleaned up, or maybe removed entirely.